core/src/clients/twitter/interactions.ts

```diff
@@ -104,15 +104,28 @@ export class TwitterInteractionClient extends ClientBase {
             ).tweets;
 
             // de-duplicate tweetCandidates with a set
-            const uniqueTweetCandidates = [...new Set(tweetCandidates)];
+            let uniqueTweetCandidates = [...new Set(tweetCandidates)];
 
             // Sort tweet candidates by ID in ascending order
-            uniqueTweetCandidates
+            uniqueTweetCandidates = uniqueTweetCandidates
                 .sort((a, b) => a.id.localeCompare(b.id))
                 .filter((tweet) => tweet.userId !== this.twitterUserId);
 
             // for each tweet candidate, handle the tweet
             for (const tweet of uniqueTweetCandidates) {
+                // Check the time is not older than 5 days.
+                const currentTime = new Date().getTime();
+                const tweetTime = new Date(tweet.timestamp * 1000).getTime();
+                const fiveDaysAgo = currentTime - 5 * 24 * 60 * 60 * 1000;
+
+                if (tweetTime < fiveDaysAgo) {
+                    console.log(
+                        "Tweet is older than 5 days, skipping tweet",
+                        tweet.id
+                    );
+                    continue;
+                }
+                
```

Comment:
Great work - Can we make this configurable based off of each character. Adding a magic number will cause issues down the road. Please add this to the character.json under the settings and read from there and default to 5 days if nothing is passed in. Thanks so much!